src/test/java/com/puppycrawl/tools/checkstyle/internal/XDocsPagesTest.java

```diff
@@ -583,6 +584,7 @@ private static String getCheckPropertyExpectedTypeName(Class<?> clss, Object ins
         final String instanceName = instance.getClass().getSimpleName();
         String result = null;
 
+        // noinspection IfStatementWithTooManyBranches
```

Comment:
yes, please create. I still believe that code should be clean, event it is a test.
Comments should be only smth that user have to read.
Nobody read javadoc, so any pollution in javadoc is ok.